.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+  "name": "C# (.NET)",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/dotnet:1-8.0-bookworm",
```

Comment:
We should just use the 8.0 SDK image here.

Reply:
The main differences currently between the devcontainers image and the official dotnet SDK image is that 
* NuGet packages will get xmldocs in the devcontainers image (which we want), and
* The devcontainers image includes Powershell Core

I think those are enough of a value-add (especially the XMLDocs) to justify using this base image.

Reply:
Educate me -- why doesn't the SDK image get the xmldocs?

RE: Powershell -- easy add-on as a feature (which is in this as well)

No objections really either, just felt like the actual SDK image felt like a better base -- it's what I use exclusively and just add features.

Reply:
> Educate me -- why doesn't the SDK image get the xmldocs?

https://github.com/dotnet/dotnet-docker/issues/2790

TLDR is "performance".

Reply:
Funnily enough, I was just going to [the source](https://github.com/dotnet/dotnet-docker/blob/main/src/sdk/8.0/bookworm-slim/amd64/Dockerfile#L14) and noticed that powershell core is already on the SDK images - so that removes one of the benefits of the devcontainers image. Kinda feels like we're all over the place with these images, thematically :)

Reply:
Yep, noticed that too -- this devcontainer image looks to take a patch to the powershell also though, which i would expect the powershell feature latest to have also.  Anyhow, thanks for the edu on the xmldocs -- seems odd our <cough> sdk image </cough> doesn't really match the sdk, but at least now I know!

Reply:
> seems odd our sdk image doesn't really match the sdk

It was super surprising to me too when I learned this too. I understand the constraints and get it adds a lot of value to the "non-human-interactive" use cases. So it seems when a human is involved, its best to start with the dev container image.